content/webapp/services/prismic/fetch/exhibitions.ts

```diff
@@ -8,7 +8,7 @@ import {
   ExhibitionPrismicDocument,
   ExhibitionRelatedContentPrismicDocument,
 } from '../types/exhibitions';
-import { Query } from '@prismicio/client';
+import * as prismic from '@prismicio/client';
```

Comment:
duplicate from line 13. interesting that { Query } was imported separately.

Reply:
Yup, just fixed – the build was upset about it.

Reply:
Same time! And yeah I see that's done in a few places too, I'll review as it's building!